packages/amplication-server/src/core/pluginInstallation/pluginInstallation.service.ts

```diff
@@ -93,13 +93,12 @@ export class PluginInstallationService extends BlockTypeService<
       user
     );
 
-    await this.analytics.track({
-      userId: user.account.id,
+    await this.analytics.trackWithContext({
       event: EnumEventType.PluginInstall,
       properties: {
         pluginId: newPlugin.pluginId,
         pluginType: "official",
-        $groups: { groupWorkspace: user.workspace.id },
+        resourceId: resource.connect.id,
```

Comment:
Not sure I understand why resourceId was added here

Reply:
to enhance tracking data. more details to understand which resource was the plugin install event was related to